content/articles/2017-12-29-dbplyr-1-2.Rmd

```diff
@@ -0,0 +1,129 @@
+  author: Giulia Bertelli
+---
+
+```{r setup, include = FALSE}
+library(dbplyr)
+library(dplyr)
+library(DBI)
+library(tibble)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We are very excited to announce that [dbplyr 1.2.0](https://cran.rstudio.com/web/packages/dbplyr/index.html)  is now available on CRAN!  
+
+The dbplyr package is the database backend for dplyr. It interacts with databases directly by translating the dplyr verbs into SQL queries. Other advantages of this approach are:
+
+<img src = "/images/dbplyr_1-2/interact.png" width = 400, align = "right">
+
+
+**Run data exploration routines over all of the data**, instead of subsets or samples.
+
+**Use the SQL Engine to run the data transformations**. In effect, dplyr pushes the computation to the database.
+
+**Collect into R only a targeted dataset.**
+
+**All of your code is in R! **  Because dplyr communicates directly with the database, there is no need to alternate between languages or tools.
+
+dbplyr is part of an ecosystem of packages meant to help analysts access and analyze data in databases. To learn more, visit our website dedicated to *all things R and databases*: http://db.rstudio.com/overview
+
+Install the latest version of dbplyr with:
+```{r, eval =  FALSE}
+install.packages("dbplyr")
+```
+
+## What's new?
+
+Here are some highlights of new features and fixes found in this release.  To see the full details please refer to the [release notes](https://github.com/tidyverse/dbplyr/blob/master/NEWS.md#dbplyr-120):
+
+- New custom translation for **Microsoft Access** and [Teradata](http://db.rstudio.com/databases/teradata/) 
+
+- [Amazon Redshift](http://db.rstudio.com/databases/redshift/) connections now point to the PostgreSQL translation.
+
+- Adds support for **two new database R packages**.  These new packages are fully [DBI](http://db.rstudio.com/dbi/)-compliant and tested with [DBItest](https://cran.r-project.org/web/packages/DBItest/index.html). We recommend to use these instead of older packages:
+    1. [RMariaDB](https://cran.r-project.org/web/packages/RMariaDB/index.html), use in favor of RMySQL
+    1. [RPostgres](https://cran.r-project.org/web/packages/RPostgres/index.html), use in favor of RPostgreSQL
+
+- [ROracle](https://cran.r-project.org/web/packages/ROracle/index.html) connections now point to the [Oracle](http://db.rstudio.com/databases/oracle/) translation.
+- Cache query results using `copy_to`. This feature is very useful because after cached, the results of a long running query can be iteratively analyzed without having to wait for the same base query to run over and over.
+
+- `copy_to()` now allows to take the results from one database and cache them in a different database.
+
+- stringr functions: `str_length()`, `str_to_upper()`, `str_to_lower()`, `str_replace_all()`, `str_detect()`, and  `str_trim()` are now supported.
+
+- `in_schema()` should now work in more places, particularly in `copy_to()` 
```

Comment:
Add brief description/usage example?

Reply:
This one is straight from the NEWS doc, but now that I'm working on an example, I can't find what has changed in the code that would be a significant impact this release. The only thing I found is the `auto_disconnect` change, it looks like `in_schema()` has worked with `copy_to()` since it's inception: https://github.com/tidyverse/dbplyr/commit/7242d10971d464b518c3a98c4982b9851ab9df09#diff-c283e9c17aedb8f5d9e01b7aa13c4854

Did I miss something?

Reply:
The issue is https://github.com/tidyverse/dplyr/issues/3013 — but maybe this is more of a bug fix, so it doesn't need more details.